test/draw/gpu/FillTest.ooc

```diff
@@ -33,7 +33,7 @@ FillTest: class extends Fixture {
 			gpuImage fill(color)
 			cpuImage fill(color)
 			gpuToCpuImage := gpuImage toRaster()
-			expect(gpuToCpuImage distance(cpuImage), is equal to(0.0f))
+			expect(gpuToCpuImage distance(cpuImage), is less than(0.05f))
```

Comment:
That would be nice but some test cases differ in tolerance so it would become a bit inconsistent.